gui/additionsPane.py

```diff
@@ -79,9 +81,12 @@ def __init__(self, parent):
         self.gangPage = GangView(self.notebook)
         self.notebook.AddPage(self.gangPage, "Fleet", tabImage = gangImg, showClose = False)
 
+        self.notes = NotesView(self.notebook)
+        self.notebook.AddPage(self.notes, "Notes", tabImage = notesImg, showClose = False)
+
         self.notebook.SetSelection(0)
 
-    PANES = ["Drones", "Fighters", "Cargo", "Implants", "Boosters", "Projected", "Fleet"]
+    PANES = ["Drones", "Fighters", "Cargo", "Implants", "Boosters", "Projected", "Fleet", "Notes"]
```

Comment:
It's quite easy to view previous blames by using git references. I do it all the time: https://github.com/resinneublem/Pyfa/blame/1ae509e189d455369eef39327c712e0fe81b5e91%5E/gui/additionsPane.py

`1ae509e189d455369eef39327c712e0fe81b5e91^` is the key